### PR TITLE
[VEN-2002] Update VAI interest rate params

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR introduces a VIP that will update will update interest rate parameters of VAI relevant to Steakhouse [recommendation](https://community.venus.io/t/vip-changing-the-interest-rate-parameters-for-vai/3735)
    * Set VAI Base Rate: 3% (from 4%)
    * Set Float Rate: 4000% (from 225%)

Resolves VEN-2002

### CODE REVIEW

**Review by chechu — APPROVED** [review]

File: vips/vip-176.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import { ProposalType } from "../src/types";
+import { makeProposal } from "../src/utils";
+
+export const VAI_CONTROLLER_PROXY = "0x004065D34C6b18cE4370ced1CeBDE94865DbFAFE";
+export const OLD_BASE_RATE_MANTISSA = parseUnits("0.04", 18); // Old Base Rate 4%
+export const NEW_BASE_RATE_MANTISSA = parseUnits("0.03", 18); // New Base Rate 3%
+
+export const OLD_FLOAT_RATE_MANTISSA = parseUnits("2.25", 18); // New Base Rate 225%
+export const NEW_FLOAT_RATE_MANTISSA = parseUnits("40", 18); // New Base Rate 4000%
```
Comment (kkirka) [inline vips/vip-176.ts:11]:
```suggestion
export const OLD_FLOAT_RATE_MANTISSA = parseUnits("2.25", 18); // Old Float Rate 225%
export const NEW_FLOAT_RATE_MANTISSA = parseUnits("40", 18); // New Float Rate 4000%
```

File: simulations/vip-176/simulations.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+forking(32052800, () => {
+  const provider = ethers.provider;
+  let vaiControllerProxy: ethers.Contract;
+
+  before(async () => {
+    vaiControllerProxy = new ethers.Contract(VAI_CONTROLLER_PROXY, VAI_CONTROLLER_ABI, provider);
+  });
+
+  describe("Pre-VIP behavior", () => {
+    it("Verify VAI base rate is 3%", async () => {
```
Comment (kkirka) [inline simulations/vip-176/simulations.ts:26]:
```suggestion
    it("Verify VAI base rate is 4%", async () => {
```

File: simulations/vip-176/simulations.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  testVip("VIP-176 Set VAI risk parameters", vip176(), {
+    callbackAfterExecution: async (txResponse: TransactionResponse) => {
+      await expectEvents(txResponse, [VAI_CONTROLLER_ABI], ["NewVAIBaseRate", "NewVAIFloatRate"], [1, 1]);
+    },
+    proposer: "0xc444949e0054a23c44fc45789738bdf64aed2391",
+    supporter: "0x55A9f5374Af30E3045FB491f1da3C2E8a74d168D",
+  });
+
+  describe("Post-VIP behavior", async () => {
+    it("Verify new VAI base rate is 4.00%", async () => {
```
Comment (kkirka) [inline simulations/vip-176/simulations.ts:45]:
```suggestion
    it("Verify new VAI base rate is 3.00%", async () => {
```

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/176

**Pull request merged**